spec/lib/eventbrite/event_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "eventbrite/event"
+require "eventbrite/venue"
+
+describe Eventbrite::Event do
+  describe "#dates" do
```

Comment:
Was this just a missing test? I don't see where this code was written in this PR.